py3status/modules/net_iplist.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Display the list of current IPs. This excludes loopback IPs and displays
+"no connection" if there is no connection.
+
+Configuration parameters
+    separator: string to use between IPs.
+        (default: ' ')
+    no_connection: string to display if there are no non-ignored IPs
+        (default: 'no connection')
+    ignore_prefix: list of prefixes to ignore.
+        (default: ['127.'])
```

Comment:
Hmm,  think I'd be inclined to do ignoring as reg expressions (althought all the `.` make it a pain)  maybe use wild cards so `ignore = ['127.*', '192.168.*.100']` type thing to be honest I've not got too many Ips on my machines to worry about this too much.